src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1721,6 +1733,42 @@ From there, simply click on a Markdown file, PDF file or SVG icon in the File Ex
     <value>Do not activate when Game Mode is on</value>
     <comment>"Game mode" is the Windows feature to prevent notification when playing a game.</comment>
   </data>
+  <data name="MouseUtils_MouseHighlighter.Header" xml:space="preserve">
+    <value>Mouse Highlighter</value>
+    <comment>Refers to the utility name</comment>
+  </data>
+  <data name="MouseUtils_Enable_MouseHighlighter.Header" xml:space="preserve">
+    <value>Enable Mouse Highlighter</value>
+    <comment>"Find My Mouse" is the name of the utility.</comment>
+  </data>
+  <data name="MouseUtils_Enable_MouseHighlighter.Description" xml:space="preserve">
+    <value>Press a shortcut to enter/exit a mode highlighting mouse clicks.</value>
+    <comment>Mouse is the hardware mouse.</comment>
+  </data>
+  <data name="MouseUtils_MouseHighlighter_LeftButtonClickColor.Header" xml:space="preserve">
+    <value>Left Button Click Color</value>
```

Comment:
We use sentence case vs. Title Case :):

_Left-button highlight color_

Reply:
Thank you. I think it's "Left button" and not "Left-button", though.